app/views/content/blog/top-tips-for-returning-teachers.md

```diff
@@ -45,9 +45,9 @@ It’s also a good idea to read up on the latest [Teachers’ Standards](https:/
 >
 > _Lara Griffiths, returning maths teacher_
 
-## Teaching maths, physics or languages? Get free one-to-one support
+## Get free one-to-one support
 
-If you hold [Qualified Teacher Status](https://www.gov.uk/guidance/qualified-teacher-status-qts) to teach maths, physics or languages, you can access a wealth of personalised guidance from a [Return to Teaching Adviser](https://adviser-getintoteaching.education.gov.uk/) (RTTA).
+If you have [qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) and want to return to teach maths, physics, chemistry, computing or languages, you can get a [Return to Teaching Adviser](https://adviser-getintoteaching.education.gov.uk/) (RTTA). They can provide a wealth of personalised guidance.
```

Comment:
Rewriting this a bit makes sense. I wonder whether the last sentence hangs a bit though?

Would it make more sense to say something like: 'you can get personalised guidance from a....'